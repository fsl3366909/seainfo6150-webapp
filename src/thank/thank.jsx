import React from 'react'

import { Switch, Route, Link } from "react-router-dom";

const Home = () => {
    return (
        <div>
            <h1>Thank you!+</h1>
            <img className='pngblock' src="https://picsum.photos/600/400" alt="" />
            <br />
            <br />
            <br />
            <br />
        </div>
    )
}

export default Home
