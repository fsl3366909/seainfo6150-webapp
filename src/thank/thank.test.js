import React from "react";
import thank from "./thank";

describe("Bar tests", () => {
    it("renders correctly", () => {
        const { container } = render(<thank categoryId={0} productId="baz" />);
        expect(container).toMatchSnapshot();
    });
});
