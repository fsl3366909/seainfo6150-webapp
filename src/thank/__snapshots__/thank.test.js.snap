// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bar tests renders correctly 1`] = `
<div>
  <thank
    categoryid="0"
    productid="baz"
  />
</div>
`;
